src/components/Practice/PhrasesInAction.jsx

```
@@ -0,0 +1,1147 @@
+  };
+
+  const playAudio2 = () => {
+    const audioElement = new Audio(levelData?.audio);
+    audioElement.play();
+  };
+
+  // Step 3
+  const [selectedWords, setSelectedWords] = useState([]);
+  const [shuffledWords] = useState(
+    currentSteps === "step3"
+      ? [...levelData?.allsentence].sort(() => Math.random() - 0.5)
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)